nservicebus/sqlserver/deployment-options_multi-instance_sqlserver_[,3].partial.md

```diff
@@ -1,8 +1,11 @@
 ### Multi-instance
 
-WARNING: The *multi-instance* option won't be supported in Versions 4 and higher. Version 3 supports *multi-instance* mode only with DTC.
+WARNING: The *multi-instance* option won't be supported in Versions 4 and higher.
 
- * Requires DTC, or in case of Version 2 using Outbox and storing business data in the same database as Outbox data.
+WARNING: Version 3 supports *multi-instance* mode with DTC only. Outbox is not supported by Version 3 in *multi-instance* mode.
```

Comment:
>WARNING: Version 3 supports *multi-instance* mode with DTC only. Outbox is not supported by Version 3 in *multi-instance* mode.

How about:

WARNING: Although Version 3 offers support for *multi-instance* mode when using DTC, Outbox is not a supported option when running in *multi-instance* mode.